### PR TITLE
(DOCSP-22152): Replace forEach with for...of for improved performance

### PR DESCRIPTION
**Pull request by mongomoe**
### Jira

- https://jira.mongodb.org/browse/DOCSP-22152

### Staged Changes (Requires MongoDB Corp SSO)

- [Read & Write Data - Node.js SDK # Bulk Update a Collection](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-22152-replace-forEach-with-forOf/sdk/node/examples/read-and-write-data/#bulk-update-a-collection)
- [Read & Write Data - React Native SDK # Bulk Update a Collection](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-22152-replace-forEach-with-forOf/sdk/react-native/examples/read-and-write-data/#bulk-update-a-collection)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/node/examples/read-and-write-data.txt [hunk 1]
```diff
@@ -176,8 +176,8 @@ Bulk Update a Collection
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
 To apply an update to a collection of objects, iterate through the collection
-(e.g. with :mdn:`forEach()
-<Web/JavaScript/Reference/Global_Objects/Array/forEach>`). In the iterator
+(e.g. with :mdn:`for..of
```
Comment (MongoCaleb) [inline source/sdk/node/examples/read-and-write-data.txt:179]: need 3 dots here: for...of

File: source/sdk/react-native/examples/read-and-write-data.txt [hunk 2]
```diff
@@ -176,8 +176,8 @@ Bulk Update a Collection
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
 To apply an update to a collection of objects, iterate through the collection
-(e.g. with :mdn:`forEach()
-<Web/JavaScript/Reference/Global_Objects/Array/forEach>`). In the iterator
+(e.g. with :mdn:`for..of
```
Comment (MongoCaleb) [inline source/sdk/react-native/examples/read-and-write-data.txt:179]: need 3 dots here: for...of

**Review by MongoCaleb — APPROVED** [review]
LGTM -- but fix the ... issue

File: source/sdk/node/examples/read-and-write-data.txt (hunk 1 above)
Comment (mongomoe) [inline source/sdk/node/examples/read-and-write-data.txt:179]:
```suggestion
(e.g. with :mdn:`for...of
```

File: source/sdk/react-native/examples/read-and-write-data.txt (hunk 2 above)
Comment (mongomoe) [inline source/sdk/react-native/examples/read-and-write-data.txt:179]:
```suggestion
(e.g. with :mdn:`for...of
```

**Pull request merged**